server/models/schema.sql

```diff
@@ -1978,6 +1978,17 @@ CREATE TABLE `stock_requisition_item` (
   CONSTRAINT `stock_req_item__stock_req_item` FOREIGN KEY (`requisition_uuid`) REFERENCES `stock_requisition` (`uuid`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
+DROP TABLE IF EXISTS `stock_adjustment_log`;
+CREATE TABLE `stock_adjustment_log` (
+  `document_uuid` BINARY(16) NOT NULL,
+  `log_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `text` JSON,
```

Comment:
Let's call this `data`, since it's JSON format.  Or `changes` or something more description.  We use `text` a lot for labels and this isn't a label.